modules/extension/wmts/src/test/java/org/geotools/ows/wmts/WebMapTileServerTest.java

```diff
@@ -16,21 +16,27 @@
  */
 package org.geotools.ows.wmts;
 
-import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertNotNull;
-import static junit.framework.TestCase.assertNull;
+import static junit.framework.TestCase.*;
 import static org.geotools.ows.wmts.WMTSTestUtils.createCapabilities;
 
+import java.io.PrintStream;
+import java.util.Set;
 import org.geotools.geometry.GeneralEnvelope;
+import org.geotools.geometry.jts.ReferencedEnvelope;
 import org.geotools.ows.wms.Layer;
 import org.geotools.ows.wmts.model.WMTSCapabilities;
+import org.geotools.ows.wmts.model.WMTSLayer;
+import org.geotools.ows.wmts.request.GetTileRequest;
 import org.geotools.referencing.CRS;
+import org.geotools.tile.Tile;
 import org.junit.Test;
 import org.opengis.referencing.crs.CoordinateReferenceSystem;
 
 /** @author Emanuele Tajariol (etj at geo-solutions dot it) */
 public class WebMapTileServerTest {
 
+    private static PrintStream out = System.out;
```

Comment:
Tests should not be writing to system.out, use the logging framework instead, or if you need it for development, keep the line that does the printout commented out, using System.out directly.

Reply:
Removed all print statements. I only needed them for development and no need for them to be left in and commented out